crates/subspace-proof-of-space/src/lib.rs

```diff
@@ -9,7 +9,8 @@
     const_trait_impl,
     generic_const_exprs,
     int_roundings,
-    iter_collect_into
+    portable_simd,
+    step_trait
 )]
 
 #[cfg(feature = "chia")]
```

Comment:
Is this intended that this and below are both "chia"?

Reply:
Yes, both modules are for Chia implementation only and `#[cfg(feature = "chia")]` only affects a single statement that follows.